dynamixel_controller_node:
  ros__parameters:
    ttl_ids: [3, 4, 5, 6]
    rs485_ids: [1]
